README.md

```diff
@@ -56,6 +56,17 @@ Here are a couple of examples of what should and should not be included:
 
 For proposing new tools, please open a Pull Request.
 
+### Commit guidelines
+
+For the general guidelines on making PRs/commits easier to review, please check
+out Kinvolk's
+[contribution guidelines on git](https://github.com/kinvolk/contribution/tree/master/git).
```

Comment:
i've just opened a PR to the contribution repo to make that directory into `./topics/git.md` instead, so maybe just have the URL be to the repo itself.